### PR TITLE
Improve Prefab Overrides Docs

### PR DESCRIPTION
**Pull request by AMZN-Gene**
Change list:
1. Reworked the workflow to make it easier to read and follow. It now treats Prefab Overrides feature as a whole thing (previously the content is split in Outliner and Inspector, which is a detail for developers)
1. Added sub headers for "Types of overrides" and "Revert an override edit"
1. Added an overview description for prefab overrides
1. Improved the "Disable prefab overrides" section with more info and directed them to known issue lists and to create a new GHI

### CODE REVIEW

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 1]
```diff
@@ -10,55 +10,84 @@ When a prefab is open in [Prefab Edit Mode](/docs/learning-guide/tutorials/entit
...
 ## Prerequisites
 
 * Basic knowledge of working with [O3DE Editor](/docs/user-guide/editor).
 * A project built from the standard project template or one that contains the Gems in the standard template.
 
+
 ## What are prefab overrides?
 
-Because prefabs instances reference a file, all instances of a prefab are identical by default. Prefab overrides allow the content of prefab instances to be modified on an individual basis.
+Prefab overrides are one of the major components of the prefab system. Before overrides, the only way to modify prefabs is by entering Prefab Edit Mode but changes would be applied to all instances of that prefab. With overrides, we can make prefabs stand out from the rest of their siblings, which gives content creators much more control and flexibility in building large and complex scenes. It also opens doors for easily overriding default procedural prefabs that are imported from `.fbx` files.
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:24]:
```suggestion
Prefab overrides are one of the major components of the prefab system. Before overrides, changing a single prefab instance would affect all instances. With overrides, we can make prefabs stand out from the rest of their siblings. Overrides give content creators more flexibility to build large and complex scenes. It also allows for overriding default procedural prefabs imported via `.fbx` files.
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 2]
```diff
@@ -10,55 +10,84 @@ When a prefab is open in [Prefab Edit Mode](/docs/learning-guide/tutorials/entit
...
+Overrides applied to prefab instances are registered for that individual instance and are stored in the prefab being edited that contains the instances. In the above example, the Car instances are in a level (which is a prefab), so the prefab overrides applied to the Car instances are stored in the level when you save it.
 
-Suppose that the car's four tires are instances of a tire prefab that are nested within the Car prefab. If you apply overrides to each of the tire instances to give them unique appearances, the overrides are stored in the Car prefab when you save it.
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/level-prefab-edit.png" width="300" alt="Level in Prefab Edit Mode with Override Edit in Entity Outliner." >}}
+
+Suppose that you modify the Body entity's location as an override from the level prefab, you notice that a blue circle will appear on the entity's icon in Entity Outliner. This indicate that an override has been applied to the entity. Now, if you save the level prefab, the override will be stored.
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:38]:
```suggestion
Modifying the Body entity's location causes a blue circle to appear in the Entity Outliner. This indicates an entity override. Now, if you save the level prefab, the override is also saved.
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 3]
```diff
@@ -10,55 +10,84 @@ When a prefab is open in [Prefab Edit Mode](/docs/learning-guide/tutorials/entit
...
 
 {{< note >}}
 A level is a prefab, and automatically enters Prefab Edit Mode when it is opened. This is indicated by the blue capsule around the level in **Entity Outliner**.
 
 Overrides are not limited to the level. In fact, any prefab that is open for editing in Prefab Edit Mode is responsible for storing the overrides made to its nested prefabs.
 {{< /note >}}
 
+However, if you do not want to make override changes, you should follow the old Prefab Edit Mode workflow instead. In the example, you can enter the mode on the Car instance before editing the Body entity, in which any changes will be applied to all Car instances as shown in the below image.
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:46]:
```suggestion
If you do not want to make override changes, you should follow the old Prefab Edit Mode workflow instead. In this example, you can enter Prefab Edit Mode via the Car instance in the Entity Outliner. Any changes to Car will now affect all Car instances.
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 4]
```diff
@@ -10,55 +10,84 @@ When a prefab is open in [Prefab Edit Mode](/docs/learning-guide/tutorials/entit
...
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-component-property.png" width="450" alt="Overriding a component property." >}}
+
+### Add or remove components
+
+To add a component of an entity under a prefab instance:
+
+1. In Entity Outliner, select an entity under an instance that is not in Prefab Edit Mode.
+1. In Inspector, **right-click** on empty area and choose **Hi** from the context menu.
+1. Notice that a blue plus icon appears on the component's icon to the left or the component name.
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:82]:
```suggestion
1. Notice that a blue plus icon appears on the component's icon to the left of the component name.
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 5]
```diff
@@ -68,94 +97,99 @@ In the below image, an Antenna entity has been added to the first Car prefab ins
...
-Alternatively, You can revert a deletion by pressing CTRL+Z to go back to a previous state where you delete that object/entity. One important note, however. This undo will only work if the editor has not been closed and reopened.
-After closing the editor, your undo stack has been cleared, and you can no longer undo in this manner.
+### Remove nested prefab instances
+
+In current workflow, it does not support adding a nested prefab instance as an override. However, you can remove a nested prefab instance as an override by following the steps mentioned above for an entity.
 
 {{< note >}}
-You cannot revert the deletion by reverting overrides on the parent entity. See the GitHub issue [#13437](https://github.com/o3de/o3de/issues/13437) for more details.
-{{< /note >}}
+It is important to note that deleting or undoing a prefab instance will not actually delete the actual prefab from the hard drive, which is the same when removing a prefab from the Entity Outliner.
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:115]:
```suggestion
Deleting or undoing a prefab instance will not delete the actual prefab off the hard drive.
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 6]
```diff
@@ -68,94 +97,99 @@ In the below image, an Antenna entity has been added to the first Car prefab ins
...
+1. With a component selected, press **DEL**.
 
-{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-add-component-override.png" width="450" alt="Adding a component as an override." >}}
+For a component deleted as an override, you can revert the deletion by reverting all overrides on the selected entity. However, be aware that this would remove other overrides if they are present.
 
-### Revert property overrides
+### Revert all overrides on the entity
 
-With the inspector override support, you can now revert overridden property values respectively. Unlike reverting overrides on an entity, you can selectively revert the property you want:
+Last but not least, you can revert all overrides that is applied on an entity:
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:153]:
```suggestion
Last but not least, you can revert all overrides applied to an entity:
```

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 7]
```diff
@@ -68,94 +97,99 @@ In the below image, an Antenna entity has been added to the first Car prefab ins
...
 
-1. In Entity Inspector, **right-click** on the property label or the blue circle and choose **Revert override** from the context menu.
-1. Notice that the property no longer has any indication of having an override.
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert.png" width="300" alt="Reverting overrides on an entity." >}}
 
-{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert-component-property-override.png" width="450" alt="Reverting a component property override." >}}
 
 {{< note >}}
-Once a property value is overridden to non-default value, the override edit can only be removed by doing a revert. In other words, setting the value back to default does not remove the override edit.
+If you see any unexpected overrides showing up, you can try the above methods to revert them. If it does not work out, you can open the prefab file that owns those overrides (under the `Instances` key) and remove the override patches you want.
```
Comment (AMZN-Gene) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:161]: pro-tip! <3

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md (hunk 3 above)
Comment (forkercat) [reply content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:46]: Love the paraphrasing!

File: content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md [hunk 8]
```diff
@@ -68,94 +97,99 @@ In the below image, an Antenna entity has been added to the first Car prefab ins
...
 {{< /note >}}
 
-In addition, you are able to revert all property overrides that are owned by a component:
 
-1. In Entity Inspector, **right-click** on the blue circle in the component header and choose **Revert Overrides** from the context menu.
-1. Notice that all the properties no longer have any indication of having an override.
+## Disable prefab overrides
 
-{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert-component-override.png" width="450" alt="Reverting a component added as an override." >}}
+Prefab overrides are a new feature. It is possible that you may encounter blocking issues that slow down your workflow. You can follow the recommendations below.
```
Comment (forkercat) [inline content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md:167]: Redundant:

```suggestion
Prefab overrides are a new feature. You may encounter issues that slow down or even block your workflow. You can follow the recommendations below to disable the feature in your project.
```

**Review by forkercat — APPROVED** [review]
Approved by myself :)

Would like to wait for @alexmontAmazon's approval before merging.

**Review by alexmontAmazon — APPROVED** [review]
looks good to me, nice update!

**Comment by forkercat** [comment]
Cannot see the page preview link. Double checked the image link and pushed a fix.

**Pull request merged**